module/entities/helper.js

```diff
@@ -1187,8 +1201,119 @@ module.exports = class EntitiesHelper {
         }
     }
 
+     /**
+   * Push entities data to elastic search
+   * @method
+   * @name pushEntitiesToElasticSearch
+   * @name entityId - created or modified entity data.
+   * @returns {Object} 
+   */
+
+  static pushEntitiesToElasticSearch(entityId) {
```

Comment:
@deepakr123 - Rename it to entities, default value to be a blank array

Reply:
resolved